admin/config-ui/class-configuration-page.php

```diff
@@ -33,7 +33,10 @@ public function __construct() {
 	 * Check if the configuration is finished. If so, just remove the notification.
 	 */
 	public function catch_configuration_request() {
-		if ( filter_input( INPUT_GET, 'configuration' ) !== 'finished' ) {
+		$configuration_page = filter_input( INPUT_GET, 'configuration' );
+		$page          = filter_input( INPUT_GET, 'page' );
+
+		if ( ! ( $configuration_page === 'finished' && ( $page === WPSEO_Configuration_Page::PAGE_IDENTIFIER ) ) ) {
```

Comment:
This should be the identifier of the `WPSEO_Admin ` because after finishing the wizard the user will be redirected to the dashboard.